custom_components/elasticsearch/es_doc_creator.py

```diff
@@ -29,6 +29,17 @@
 from custom_components.elasticsearch.system_info import SystemInfo
 
 ALLOWED_ATTRIBUTE_TYPES = tuple | dict | set | list | int | float | bool | str | None
+SKIP_ATTRIBUTES = [
+    "friendly_name",
```

Comment:
`friendly_name` !== `hass.entity.name` in all cases: https://developers.home-assistant.io/docs/core/entity#entity-naming

Reply:
oops i actually thought we already had a field for friendly_name

we could:
1. Add entity.friendly_name
2. update entity.name to follow the logic in the documentation
3. leave it as an attribute that exists on almost every single document

Reply:
updated to do 1 but will make this a 2.0 change